cibuild.cmd

```diff
@@ -107,3 +100,14 @@ exit /b 0
 @echo            with /release. May not be combined with /test32 or /test64.
 @echo.
 @goto :eof
+
+:BuildFailed
+echo Build failed with ERRORLEVEL %ERRORLEVEL%
+call :TerminateCompilerServer
+exit /b 1
+
+:TerminateCompilerServer
+@REM Kill any instances VBCSCompiler.exe to release locked files, ignoring stderr if process is not open
+@REM This prevents future CI runs from failing hile trying to delete those files.
+
+taskkill /F /IM vbcscompiler.exe 2> nul
```

Comment:
Yeah, it's interesting - I like someone else to comment on this, I haven't changed the logic here. I'd rather tell the compiler server not to reuse nodes, that seems a lot cleaner.